app/views/about/show.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= render partial: '/partials/topbar/topbar-secondary', locals: { links: header_nav } %>
+<%= render partial: '/partials/topbar/topbar-secondary', locals: { links: header_nav(current_user) } %>
 
 <%= render partial: '/partials/hero/hero_about', locals: { title: I18n.t('about.hero') } %>
```

Comment:
Not really this PR, but just spotte here that the partial is snake cased, whereas the others are kebab

Reply:
changed to camel case